docs/installation.md

```diff
@@ -1,83 +1,127 @@
-        ```sh
-        sudo curl -L https://bintray.com/odo/odo-rpm-releases/rpm -o /etc/yum.repos.d/bintray-odo-odo-rpm-releases.repo
-        ```
-3. Now you can install `odo` and you would install any other package.
-   ```sh
-   yum install odo
-   # or 'dnf install odo'
-   ```
+
+#### Enable via Homebrew Tap
+
+In order to access the repo:
+
+```sh
+brew tap kadel/odo
+```
+
+#### Install
+
+The latest MASTER build:
+
+```sh
+brew install kadel/odo/odo -- HEAD
+```
+
+Latest released version:
+
+```sh
+brew install kadel/odo/odo
+```
+
+## Deb (Debian / Ubuntu)
+
+#### Add GPG Key
+
+Add the GPG key from Bintray used to sign repositories:
+
+```sh
+curl -L https://bintray.com/user/downloadSubjectPublicKey?username=bintray | apt-key add -
+```
+
+Add the repository to `/etc/apt/sources.list`:
+
+```sh
+# For latest Master builds
+echo "deb https://dl.bintray.com/odo/odo-deb-dev stretch main" | sudo tee -a /etc/apt/sources.list
+
+# For latest signed releases
+echo "deb https://dl.bintray.com/odo/odo-deb-releases stretch main" | sudo tee -a /etc/apt/sources.list
+```
+
+Now install Odo:
+
+```sh
+sudo apt-get update
+sudo apt-get install odo
+```
+
+## RPM (Fedora / CentOS / RHEL)
+
+#### Add the Odo repository to `/etc/yum.repods.d/`:
+
+For latest builds:
+
+```sh
+$ vim /etc/yum.repos.d/bintray-odo-odo-rpm-dev.repo
+```
+
+```
+# /etc/yum.repos.d/bintray-odo-odo-rpm-dev.repo
+[bintraybintray-odo-odo-rpm-dev]
+name=bintray-odo-odo-rpm-dev
+baseurl=https://dl.bintray.com/odo/odo-rpm-dev
+gpgcheck=0
+repo_gpgcheck=0
+enabled=1
+```
+
+Or you can download it using following command:
+
+```sh
+sudo curl -L https://bintray.com/odo/odo-rpm-dev/rpm -o /etc/yum.repos.d/bintray-odo-odo-rpm-dev.repo
+```
+
+For the latest release:
+
+```sh
+$ vim /etc/yum.repos.d/bintray-odo-odo-rpm-releases.repo
+```
+
+```
+# /etc/yum.repos.d/bintray-odo-odo-rpm-releases.repo
+[bintraybintray-odo-odo-rpm-releases]
+name=bintray-odo-odo-rpm-releases
+baseurl=https://dl.bintray.com/odo/odo-rpm-releases
+gpgcheck=0
+repo_gpgcheck=0
+enabled=1
+```
+
+Or you can download it using following command:
+
+```sh
+sudo curl -L https://bintray.com/odo/odo-rpm-releases/rpm -o /etc/yum.repos.d/bintray-odo-odo-rpm-releases.repo
+```
+
+#### Install Odo
+
+```sh
+# CentOS / RHEL
+yum install odo
+
+# Fedora
+dnf install odo
+ ```
 
 ## Windows
-Download latest master builds from Bintray [odo.exe](https://dl.bintray.com/odo/odo/latest/windows-amd64/:odo.exe) or 
-builds for released versions from [GitHub releases page](https://github.com/redhat-developer/odo/releases).
+
+1. Download the latest  file from Bintray ([odo.exe](https://dl.bintray.com/odo/odo/latest/windows-amd64/odo.exe)) or from the latest release page on [GitHub](https://github.com/redhat-developer/odo/releases).
+2. Extract the file
+3. Add the location of extracted binary to your PATH environment variable by following [this Wiki page](https://github.com/redhat-developer/odo/wiki/Setting-PATH-variable-on-Windows).
```

Comment:
Wouldn't it be better to have everything in one place?
I would consider moving that wiki page to this document.